src/modules/aws-ec2/termination-protection/disableTerminationProtection.ts

```diff
@@ -0,0 +1,25 @@
+import { logger } from "../../../logger";
+import { RunbookStep } from "../../RunbookStep";
+import { changeTerminationProtection } from "./sub-modules/changeTerminationProtection";
+import { describeTerminationProtection } from "./sub-modules/describeTerminationProtection";
+
+export class DisableTerminationProtection extends RunbookStep {
+  async describeAction(instanceId: string): Promise<void> {
+    const currentValue = await describeTerminationProtection(instanceId);
+    const targetValue = false;
```

Comment:
This line the `targetValue` variable is `false`. In the `run` function is `"true"`. Which one is correct?

Reply:
`false` is correct.